src/install/sql/content.sql

```diff
@@ -322,28 +322,29 @@ VALUES
-	(13,'company_note','Amet auctor, sed massa lacus phasellus turpis urna mauris dictumst, dapibus turpis? Sociis amet, mid aliquam, sagittis, risus, eros porta mid placerat eros in? Elementum porta ac pulvinar porttitor adipiscing, tristique porta pid dolor elementum? Eros, pulvinar amet auctor, urna enim amet magnis ultrices etiam? Dictumst ultrices velit eu tortor aliquet, rhoncus! Magnis porttitor. Vel parturient, ac, nascetur magnis tincidunt.\n\nQuis, pid. Lacus lorem scelerisque tortor phasellus, duis adipiscing nec mid mus purus placerat nunc porttitor placerat, risus odio pulvinar penatibus tincidunt, proin. Est tincidunt aliquam vel, ut scelerisque. Enim lorem magna tempor, auctor elit? Magnis lorem ut cursus, nunc nascetur! Est et odio nunc odio adipiscing amet nunc, ridiculus magnis egestas proin, montes nunc tristique tortor, ridiculus magna.\n',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(14,'invoice_series','FOSS',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(15,'invoice_due_days','5',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(16,'invoice_auto_approval','1',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(17,'invoice_issue_days_before_expire','14',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(18,'theme','huraga',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(19,'issue_invoice_days_before_expire','7',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(20,'invoice_refund_logic','credit_note',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(21,'invoice_cn_series','CN-',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(22,'invoice_cn_starting_number','1',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(23,'nameserver_1',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(24,'nameserver_2',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(25,'nameserver_3',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(26,'nameserver_4',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(27,'funds_min_amount','10',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(28,'funds_max_amount','200',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00');
+	(7,'company_favicon','themes/huraga/assets/favicon.ico',0,NULL,NULL,'2023-01-08 12:00:00','2023-01-08 12:00:00'),
```

Comment:
Why not append it as 29?

Also missing upgrade script

Reply:
I think it's better close to the logo if it's opened via the mysql management ui

where do i find the upgrade script

Reply:
> I think it's better close to the logo if it's opened via the mysql management ui
> 
> where do i find the upgrade script

https://github.com/FOSSBilling/FOSSBilling/blob/3c793a630eacd7ce3713c49bd9c7b8cb7c5fb6eb/src/foss-update.php#L27

The location in the form is hardcoded set and not based on the order... Aand for users who are willing to change it via phpmyadmin it should not matter  they are used by the pain full way